apps/ops/api/__init__.py

```diff
@@ -4,3 +4,4 @@
 from .celery import *
 from .job import *
 from .playbook import *
+from .variable import *
```

Comment:
The main difference between this version of the code and the previous one is that it has been updated with new libraries. To optimize the code, we recommend updating dependencies to avoid unnecessary imports, as well as removing unused methods. Additionally, for consistency and future reusability, consider renaming variables and functions appropriately.